.github/workflows/main.yml

```diff
@@ -11,6 +11,22 @@ on:
   pull_request:
 
 jobs:
+  close-issues:
```

Comment:
FYI: this job can be removed now from the main workflow

Reply:
done, thanks for a nice catch!